python/pyarrow/tests/test_compute.py

```diff
@@ -1051,6 +1051,25 @@ def test_pad():
     assert pc.utf8_rpad(arr, 3).tolist() == [None, 'á  ', 'abcd']
 
 
+def test_utf8_zfill():
+    arr = pa.array(['A', 'AB', 'ABC', None])
+    assert pc.utf8_zfill(
+        arr, options=pc.PadOptions(width=3, padding='0')
```

Comment:
This could work with `pc.utf8_zfill(arr, width=3)` or `pc.utf8_zfill(arr, 3)` if I am not mistaken?